modules/data.atmosphere/R/download.AmerifluxLBL.R

```diff
@@ -92,7 +95,11 @@ download.AmerifluxLBL <- function(sitename, outfolder, start_date, end_date,
   
   if (download_file_flag) {
     extract_file_flag <- TRUE
-    utils::download.file(ftplink, output_zip_file)
+    if (method != "ncftpget") {
+    	utils::download.file(ftplink, output_zip_file)
+    } else {
+    	PEcAn.utils::download.file(ftplink, output_zip_file, method=method) 
```

Comment:
If PEcAn.utils::download.file also has to do a check against method, why have this check here? Why not just always call PEcAn.utils::download.file and then inside that function have it use utils::download.file as the default?